monsteer/brionplugin/spikeReport.h

```diff
@@ -49,32 +49,31 @@ class SpikeReport : public brion::SpikeReportPlugin
 
     const brion::Spikes& getSpikes() const final;
 
-    bool waitUntil(const float timeStamp, const uint32_t timeout) final;
+    bool waitUntil( const float timeStamp, const uint32_t timeout ) final;
```

Comment:
I don't understand your discussion... what should I do here..? Remove the const?